plugins/prompt-generator/scripts/svg-to-prompt.py

```
@@ -166,18 +167,156 @@ def canvas_strokes(strokes, bounds):
+            if combined[-1] == oriented[0]:
+                combined.extend(oriented[1:])
+            else:
+                combined.extend(oriented)
+            current_index = next_index
+        merged.append((color, combined))
+    return merged
+
+
 def main():
     parser = argparse.ArgumentParser(description=__doc__)
     parser.add_argument("--input", required=True); parser.add_argument("--output", required=True)
```

Ruff (0.16.2)

[error] Multiple statements on one line (semicolon)

(E702)